Kinvey/Kinvey/FindOperation.swift

```diff
@@ -147,145 +147,29 @@ internal class FindOperation<T: Persistable>: ReadOperation<T, AnyRandomAccessCo
-                                }
-                                self.executeLocal {
-                                    switch $0 {
-                                    case .success(let results):
-                                        fulfill(results)
-                                    case .failure(let error):
-                                        reject(error)
-                                    }
-                                }
-                                }.catch { error in
-                                    reject(error)
-                            }
-                        } else if allIds.count > 0 {
-                            let query = Query(format: "\(Entity.Key.entityId) IN %@", allIds)
-                            var newRefObjs: [String : String]? = nil
-                            let operation = FindOperation<T>(
-                                query: query,
-                                deltaSet: false,
-                                autoPagination : self.autoPagination,
-                                readPolicy: .forceNetwork,
-                                validationStrategy: self.validationStrategy,
-                                cache: cache,
-                                options: self.options
-                            ) { jsonArray in
-                                newRefObjs = self.reduceToIdsLmts(jsonArray)
-                            }
-                            operation.execute { (result) -> Void in
-                                switch result {
-                                case .success:
-                                    if self.mustRemoveCachedRecords,
-                                        let refObjs = newRefObjs
-                                    {
-                                        self.removeCachedRecords(
-                                            cache,
-                                            keys: refObjs.keys,
-                                            deleted: deltaSet.deleted
-                                        )
-                                    }
-                                    self.executeLocal {
-                                        switch $0 {
-                                        case .success(let results):
-                                            fulfill(results)
-                                        case .failure(let error):
-                                            reject(error)
-                                        }
-                                    }
-                                case .failure(let error):
-                                    reject(error)
-                                }
-                            }
-                        } else {
+            if deltaSet, let cache = self.cache {
+                let lastPull = cache.lastPull ?? Date(timeIntervalSince1970: 0)
```

Comment:
If `cache.lastPull` isn't available, we shouldn't use `Date(timeIntervalSince1970:0)`. We should fallback to regular fetch.